packages/components/src/Card/Card.css

```diff
@@ -40,19 +40,19 @@
 }
 
 .clickable {
-  --card--border: none;
-
-  box-shadow: var(--shadow-base);
   color: inherit;
   text-decoration: inherit;
   cursor: pointer;
-  transition: all var(--timing-quick) linear;
+  transition: all var(--timing-base) ease-out;
 }
 
 .clickable:hover,
 .clickable:focus {
   background-color: var(--color-surface--hover);
 }
+.clickable:focus-visible {
+  box-shadow: var(--shadow-focus);
```

Comment:
I've just realized that we're already handling the focus in the "standard" (ie leveraging `outline`) on `Card` [(up on L10)](https://github.com/GetJobber/atlantis/pull/2054/files#diff-1bf632c749827ced2b3e80addf18ec8b10759e48e8973b61c30d2fa618ae950cR10) and this now conflicts with that (particularly noticeable in a "high" elevation), as the "elevation" shadow gets replaced with the "focus" shadow. 

Example:
https://github.com/user-attachments/assets/01cef92d-4920-44d9-a1d3-54e0a6a09253

We could stack the shadows by comma-separating them like

`box-shadow: var(--shadow-focus, var(--shadow-{myCurrentElevation})` 

but I think we can actually just drop this new shadow declaration here and get more or less the same end result with much less complexity.

Reply:
I see what you mean. Always game for less complexity so I have dropped this.